lib/proportions.ts

```diff
@@ -121,7 +121,9 @@ export const Proportions = function (filterManager: ReturnType<typeof DataDistri
     let fwDict = count(nodes, ["firmware", "release"]);
     let baseDict = count(nodes, ["firmware", "base"]);
     let deprecationDict = count(nodes, ["model"], function (d) {
-      return config.deprecated && d && config.deprecated.includes(d) ? _.t("yes") : _.t("no");
+      if (config.deprecated && d && config.deprecated.includes(d)) return _.t("deprecation");
+      if (config.eol && d && config.eol.includes(d)) return _.t("eol");
```

Comment:
I just received the feedback that it would be better to call the "eol" section "end-of-life" in the statistics filter?
What do you think @grische

Reply:
I guess you mean that location?

```diff
diff --git a/public/locale/de.json b/public/locale/de.json
index 03d57ab..b361299 100644
--- a/public/locale/de.json
+++ b/public/locale/de.json
@@ -101,7 +101,7 @@
   "close": "schließen",
   "deprecation": "deprecated",
   "deprecation-text": "Warnung: Dieser Knoten ist veraltet, und wird in Zukunft nicht mehr unterstützt. Mehr Infos unter <a href='https://openwrt.org/supported_devices/864_warning'>4/32 warning</a>. <br> Wenn du der Eigentümer des Gerätes bist, bitten wir dich, das Gerät zu ersetzen, um weiterhin am Netz teilnehmen zu können.",
-  "eol": "eol",
+  "eol": "end-of-life",
   "eol-text": "Warnung: Dieser Knoten ist veraltet, und nicht mehr unterstützt. Mehr Infos unter <a href='https://openwrt.org/supported_devices/432_warning'>4/32 warning</a>. <br> Wenn du der Eigentümer des Gerätes bist, bitten wir dich, das Gerät zu ersetzen, um weiterhin am Netz teilnehmen zu können.",
   "loading": "%{name} graph (wird generiert)"
 }
diff --git a/public/locale/en.json b/public/locale/en.json
index 469fd21..3e787fe 100644
--- a/public/locale/en.json
+++ b/public/locale/en.json
@@ -101,7 +101,7 @@
   "close": "close",
   "deprecation": "deprecated",
   "deprecation-text": "This node is deprecated, and will be out of support soon. More information under <a href='https://openwrt.org/supported_devices/864_warning'>8/64 warning</a>.<br>If you're the owner, please replace it with an modern device!",
-  "eol": "eol",
+  "eol": "end-of-life",
   "eol-text": "This node has reached end-of-life, and is not supported anymore. More information under <a href='https://openwrt.org/supported_devices/432_warning'>4/32 warning</a>.<br>If you're the owner, please replace it with an modern device!",
   "loading": "%{name} graph (is generated)"
 }
```